Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM php:7.4-cli
+FROM php:8.0-cli
 
-RUN docker-php-ext-install -j$(nproc) bcmath
```

Comment:
what's the problem with using the official way of installing PHP extensions?

Reply:
I just found this method easier to install php-extensions.
I lost too much time searching the correct lib to make one extension work.

But if it's a problem for you, I can update the Docker file with `docker-php-ext-install`

Reply:
yeah, I'd try to keep using `docker-php-ext-install`